tests/api.c

```diff
@@ -155,7 +168,7 @@
     #ifndef HEAP_HINT
         #define HEAP_HINT   NULL
     #endif
-    static int devId = INVALID_DEVID;
```

Comment:
What was the reason for removing the static devId in favor of having each function declare their own devId?